pkg/services/rendering/interface.go

```diff
@@ -11,7 +11,15 @@ import (
 var ErrTimeout = errors.New("timeout error - you can set timeout in seconds with &timeout url parameter")
 var ErrPhantomJSNotInstalled = errors.New("PhantomJS executable not found")
 
+type RenderType string
+
+const (
+	RENDER_CSV RenderType = "csv"
+	RENDER_PNG RenderType = "png"
```

Comment:
Constants shouldn't be written in ALL_UPPERCASE in Go (I know a lot of the Grafana codebase thinks otherwise, but it's a thing from other languages, f.ex. C):

```suggestion
	renderCSV RenderType = "csv"
	renderPNG RenderType = "png"
```